config/setup/tls.go

```diff
@@ -25,6 +53,64 @@ func TLS(c *Controller) (middleware.Middleware, error) {
 			return nil, c.ArgErr()
 		}
 		c.TLS.Key = c.Val()
+
+		// Optional block
+		for c.NextBlock() {
+			switch c.Val() {
+			case "protocols":
+				args := c.RemainingArgs()
+				if len(args) != 2 {
+					return nil, c.ArgErr()
```

Comment:
sure, but the first is an error in number of arguments
I am changing the others =]